src/DataSource.ts

```diff
@@ -55,6 +55,21 @@ export class DataSource extends DataSourceWithBackend<TimestreamQuery, Timestrea
     return query.rawQuery ?? '';
   }
 
+  private interpolateVariable = (value: string | string[]) => {
+    if (typeof value === 'string') {
+      return value;
+    }
+
+    const quotedValues = value.map((v) => {
+      return this.quoteLiteral(v);
+    });
+    return quotedValues.join(',');
+  };
+
+  private quoteLiteral(value: any) {
```

Comment:
I know this was copied from Athena but should this value also be `string` if we are doing a map of the value that is `string[]`?

Reply:
let me verify, I believe you are right. Is it supported to have template variables for other things than strings?